lease.go

```diff
@@ -109,7 +110,7 @@ func (lm *FileLeaseManager) findNextAvailableIpAddress() (*net.IPNet, error) {
 	for _, r := range lm.wgRecords {
 		allocatedIPs = append(allocatedIPs, r.IP)
 	}
-	availableIPs, err := getAvailableIPAddresses(lm.cidr, allocatedIPs)
```

Comment:
Wouldn't it be simpler to just pass `append(allocatedIPs, lm.ip)` here?

Reply:
Yep, I like that